data/styles/clear/include/Roads.mapcss

```diff
@@ -185,19 +180,25 @@ line|z19-[highway=trunk_link]
 
 /* 3.2 Trunk & Motorway tunnel 12-22 ZOOM */
 
+line|z12-[highway=motorway][tunnel?],
+line|z12-[highway=trunk][tunnel?],
+line|z13-[highway=motorway_link][tunnel?],
+line|z13-[highway=trunk_link][tunnel?],
```

Comment:
Is it intended to have a different motorway and _link on z12?
xxx_link will have only _core_ line on z12 then ..

Reply:
Yeap
- before the change motorway link tunnels had a **solid** (instead of dashed) outline on z12 (a bug obviously)
- we start showing tunnel outlines for less important roads later, e.g. primary tunnels start at z14-, so it fine to show links' outlines later too

Reply:
Hm, I see that before the change link tunnels have dashes.
After the change only solid on z12.